libs/components/datetime/src/lib/modules/date-pipe/date-format-utility.ts

```diff
@@ -33,15 +33,15 @@ export class SkyDateFormatUtility {
     // introduce a breaking change, so we check for it here. This could probably be removed
     // in a future major version.
     if (value instanceof Object && !(value instanceof Date)) {
-      handleInvalidDate(value);
+      throw new Error('Invalid value: ' + value);
     }
 
     // Use moment to avoid inconsistencies between browsers interpreting the value differently.
     const momentDate = moment(value);
     if (momentDate.isValid()) {
       date = momentDate.toDate();
     } else {
-      handleInvalidDate(value);
+      throw new Error('Invalid value: ' + value);
```

Comment:
Why'd we get rid of the handy function? Couldn't you set the `value` parameter's type to `unknown`?

Reply:
Because the strict mode analyzer doesn't see the `date` property as being properly set if you throw the error in the function as it sees a path of code that does not set `date` and it isn't smart enough to realize that that path calls a function that throws an error.